packages/frontend/src/pages/Playground.svelte

```diff
@@ -71,6 +72,37 @@ afterUpdate(() => {
 async function scrollToBottom(element: Element) {
   element.scroll?.({ top: element.scrollHeight, behavior: 'smooth' });
 }
+
+function isHealthy(status?: string, health?: string): boolean {
+  return status === 'running' && (!health || health === 'healthy');
+}
+
+function getStatusColor(status?: string, health?: string): string {
+  return isHealthy(status, health) ? 'bg-green-600' : 'bg-gray-900';
+}
+
+function getStatusText(status?: string, health?: string): string {
+  switch (status) {
+    case 'running':
+      switch (health) {
+        case 'healthy':
+          return 'Model Service healthy';
+        default:
+          return 'Model Service not healthy';
+      }
+    default:
+      return 'Model Service not running';
```

Comment:
From a user perspective idk if it makes sense to use the term healthy. Maybe we can just show `Model Service running` - `Model Service not running` and add further explanation/hints in the tooltip (if we can have any).

Reply:
Yes, you are right. I also introduced a "Model Service starting" state, as discussed for the AI App today during UX call